sys/ztimer/auto_init.c

```diff
@@ -105,12 +105,12 @@ ztimer_clock_t *const ZTIMER_USEC = &_ztimer_convert_frac_usec.super.super;
 #endif
 
 #if MODULE_ZTIMER_MSEC
-#  if MODULE_PERIPH_TIMER_RTT
-static ztimer_periph_timer_rtt_t _ztimer_periph_timer_rtt_msec;
+#  if MODULE_PERIPH_RTT
```

Comment:
ups, I thing this happened when renaming `ztimer_(periph|rtt|rtc)` to `ztimer_periph_(timer|rtt|rtc)` with search&replace.